### DISCUSSION TITLE
My Outline container log file is HUGE

### DISCUSSION THREAD

**Discussion by jeffshead** [outline/outline#4974] [Show and tell]
I'm posting this in case someone else runs into this issue.

Twice my server ran out of space and I haven't been using Outline. It's been up and running for a few months but not being used, at all. The first time my server ran out of space, I increased the space by 20 GB thinking that it must be because I added some other containers and I added several pics and attachments in other Wiki and note taking Docker containers. Fast forward to today (a month or so after adding the 20 GB) and I ran out of space again. After doing some digging, I found that I had a single container (/var/lib/docker/containers/_long string_) that was over 20 GB. Then I found out it was the Outline container and that the log file (_long string_-json.log) itself was over 20 GB.

When I was setting things up, I added the following to my Outline stack in Portainer, in the environment section:
`- DEBUG=*`

I added that because I was having issues getting things up and running but forgot to remove it... DOH!

Long story short -- If you ever add the DEBUG environment variable, don't forget to remove it.
